File: LeetCode/Easy/1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.go
func oddCells(m int, n int, indices [][]int) int {
    matrix := make([][]int, m)
    
    for i := 0; i < m; i++ { matrix[i] = make([]int, n) }

    for _, v := range indices {
        row := v[0]
        col := v[1]

        for i := 0; i < n; i++ { matrix[row][i]++ }

        for i := 0; i < m; i++ { matrix[i][col]++ }
    }

    ret := 0

    for row := 0; row < m; row++ {
        for col := 0; col < n; col++ {
            if matrix[row][col] % 2 == 1 { ret++ }
        }
    }

    return ret
}

File: LeetCode/Easy/1252-cells-with-odd-values-in-a-matrix/1252-cells-with-odd-values-in-a-matrix.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        int[][] matrix = new int[m][n];

        for (int[] v : indices){
            int row = v[0];
            int col = v[1];
            
            for (int i = 0; i < n; ++i)
                ++matrix[row][i];
            
            for (int i = 0; i < m; ++i)
                ++matrix[i][col];
        }
        
        int ret = 0;
            
        for (int row = 0; row < m; ++row)
            for (int col = 0; col < n; ++col)
                if (matrix[row][col] % 2 == 1) ++ret;


        return ret;
    }
}